qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -968,6 +970,95 @@ def _wait_for_callbacks(self, timeout: Optional[float] = None):
                 "Possibly incomplete analysis results: an analysis callback raised an error."
             )
 
+    def monitor(self, interval=2, output=sys.stdout, line_discipline="\r"):
+        """Monitors and outputs the live status of the experiment.
+
+        The output is in format "Experiment Status:" followed by one of the possible
+            * JOB (1/N) `STATUS` if the experiment is in the job phase. For full possible statuses of
+              each job, see the :meth:`status` method.
+            * ANALYSIS IN PROGRESS if the experiment is in the analysis phase.
```

Comment:
After #599 this could be changed to something like  `Analysis (1/N) `STATUS` like for jobs, since you should be able to see how many analysis jobs there are and get the status analysis status.